tests/unit/model/test_model_item.py

```diff
@@ -18,20 +18,50 @@
 
 import pytest
 
+from structurizr.model import Model, SoftwareSystem, SoftwareSystemIO
 from structurizr.model.model_item import ModelItem
 
 
+@pytest.fixture(scope="function")
+def model() -> Model:
+    return Model()
+
+
 class ConcreteModelItem(ModelItem):
     """Implement a concrete `ModelItem` class for testing purposes."""
 
     pass
 
 
 @pytest.mark.parametrize(
-    "attributes", [{}],
+    "attributes",
+    [{}],
 )
 def test_model_item_init(attributes):
     """Expect proper initialization from arguments."""
     model_item = ConcreteModelItem(**attributes)
     for attr, expected in attributes.items():
         assert getattr(model_item, attr) == expected
+
+
+def test_get_tags_when_there_are_no_tags(model: Model):
+    element = model.add_software_system(name="Name", description="Description")
+    assert list(element.tags) == ["Element", "Software System"]
+
+
+def test_get_tags_returns_the_list_of_tags_when_there_are_some_tags(model: Model):
+    # Based on the equivalent Java test, but also checks tag ordering is preserved
```

Comment:
Please integrate this comment in a docstring for the test.

Reply:
Done.